pages/appReview.tsx

```diff
@@ -14,10 +14,15 @@ import Footer from "../components/Footer";
 import { Button, Container, Background, Flex, Column } from "../styles";
 
 const AppReview = ({ hackerProfile, reviewHistory }) => {
-  const [s1, setS1] = useState(null);
-  const [s2, setS2] = useState(null);
-  const [s3, setS3] = useState(null);
-  const [comments, setComments] = useState("");
+  const [currentProfile, setCurrentProfile] = useState(hackerProfile);
+  const [reviewCount, setReviewCount] = useState(
+    reviewHistory ? reviewHistory.length : 0
+  );
+  const [submitting, setSubmitting] = useState(false);
+
+  const [s1, setS1] = useState("");
+  const [s2, setS2] = useState("");
+  const [s3, setS3] = useState("");
```

Comment:
defaulting to blank string instead of null or undefined, if we do null/undefined and have our inputs use s1/s2/s3, we get an error about them being uncontrolled components and what-not